Answers.md

```diff
@@ -1,9 +1,15 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
+  Semantic HTML just means that you're giving meaning to your code.  For example, if you were making a navigation bar for your website, you might use <nav> instead of just <div>.  It shows the next person reading, editing, or reviewing your code what you were doing and what your intentions for that code were.
 
```

Comment:
This is a good answer, but don't forget about semantic HTML and how it relates to search engines +1